rbac/management/group/view.py

```diff
@@ -605,6 +610,16 @@ def filters_from_params(self, valid_filters, model_name, request):
                 filters[f"{attr_filter_name}__icontains"] = param_value
         return filters
 
+    def validate_guid(self, uuid):
```

Comment:
This might be great to have in `utils.py`, what do you think? Seems like we could reuse this across different views. We're doing similar here: https://github.com/RedHatInsights/insights-rbac/blob/master/rbac/management/group/view.py#L71-L76

We'd just need to update the signature to allow for customizing the key.

Reply:
Oh, yeah, that's a good idea. I'd thought about dropping the function elsewhere, but completely forgot about `utils.py`. I'll get that set up in a few